indico/modules/categories/templates/display/sidebar.html

```diff
@@ -66,9 +66,11 @@ <h2 class="icon-alarm">{% trans %}Upcoming events{% endtrans %}</h2>
         <ul class="main-page-list">
             {% for event in upcoming_events %}
                 <li>
+                    {% block upcoming_title %}
```

Comment:
Have you tested this, and also accessing the `event` inside the custom block? Pretty sure you need to use the `scoped` modifier on the block definition for this to work!

```suggestion
                    {% block upcoming_title scoped %}
```

Also make sure to test with multiple events, I'm not completely sure how blocks interact with loops. Probably they're fine but best to test this.

Reply:
"scoped" added.
Yes, I have tested with the rendering of a list go events and it works.
For info, we added a prefix to the event title when rendering the list.

Reply:
I'm very surprised it worked for you without `scoped`. When I remove the `scoped` modifier, simply accessing the indico home page (with upcoming events), it fails:

> jinja2.exceptions.UndefinedError: 'event' is undefined

Reply:
Indeed, it works for me because in the overridden template I used "scoped" (no scoped in original block, scoped used in overridden block).
Thanks for the approval.